### PR TITLE
Test fixed

### PR DESCRIPTION
**Pull request by villenavg**

### CODE REVIEW

File: tests/LOOP/test_35292.py [hunk 1]
```diff
@@ -19,51 +19,74 @@
...
             self.loop.allow_permission_phone_login()
             self.UTILS.element.waitForElements(DOM.Loop.app_header, "Loop main view")
-
         self.apps.kill_all()
         time.sleep(2)
+
         _ = setup_translations(self)
-        self.expected_message = _("No problem! Just share the following link and they can call you back from"\
-                                  " any browser.")
+        msg = "{} is not a Firefox Hello user yet. No problem! Just create a room and share it with him."
```
Comment (FranciscoCano) [inline tests/LOOP/test_35292.py]: This string is spread all over the tests, maybe it could be put somewhere in the TK

Comment (villenavg) [inline tests/LOOP/test_35292.py]: Somewheeeer oveeeer the TK :rainbow:

File: tests/LOOP/test_35297.py [hunk 2]
```diff
@@ -55,41 +51,54 @@ def setUp(self):
...
+        self.UTILS.test.test(to_address.text == self.test_contact['email']['value'], "Recipient address: {} Expected: {}".\
+                             format(to_address.text, self.test_contact['email']['value']))
         subject = self.UTILS.element.getElement(DOM.Email.compose_subject, "Email subject")
+        
         self.UTILS.test.test(subject.get_attribute("value") == "Firefox Hello", "Email subject: {} Expected:"\
                              " Firefox Hello".format(subject.get_attribute("value")))
         compose_msg = self.UTILS.element.getElement(DOM.Email.compose_msg, "Message body")
-        expected_body = _("Click on the link and answer the call! https://hello.firefox.com/#call")
+        
+        expected_body = _("""Join me for a video chat in my Firefox Hello room {}""".format(self.room_subject))
```
Comment (FranciscoCano) [inline tests/LOOP/test_35297.py]: No need to use triple quotes, but don't worry

**Comment by FranciscoCano** [comment]
LGTM

File: tests/LOOP/test_35297.py (hunk 2 above)
Comment (villenavg) [inline tests/LOOP/test_35297.py]: Residual from previous message. Sorry

**Pull request merged**